packages/ui/src/test/index.test.ts

```diff
@@ -0,0 +1,20 @@
+import { join, dirname, basename } from 'node:path';
+import { readFile } from 'node:fs/promises';
+import glob from 'fast-glob';
+
+describe( 'index', () => {
+	it( 'should export something from each component', async () => {
+		// As described in the CONTRIBUTING.md file, each component should be
+		// exported from an index.ts in its implementation directory.
+		const components = await glob( __dirname + '/../*/index.ts' );
```

Comment:
Wouldn't this still be mixed slash?

```
C:\foobar\gutenberg\packages\ui\src\test/../*/index.ts
```

Maybe `cwd` in the options? That pattern seems to be used in a few other places.

```suggestion
		const components = await glob( '*/index.ts', { 
			cwd: join( __dirname, '..' ) 
		} );
```

Reply:
> Wouldn't this still be mixed slash?

Argh, yeah, you're right.

> Maybe cwd in the options? That pattern seems to be used in a few other places.

This could work 👍